metadata-ingestion/src/datahub/ingestion/source/aws/glue.py

```diff
@@ -204,6 +206,8 @@ class GlueSourceReport(StaleEntityRemovalSourceReport):
     num_job_script_failed_parsing: int = 0
     num_job_without_nodes: int = 0
     num_dataset_to_dataset_edges_in_job: int = 0
+    num_dataset_schema_invalid: int = 0
+    num_dataset_buggy_delta_schema: int = 0
```

Comment:
Note for reviewers:
I added these two mainly for the testing. I'm ok to rename or remove even

Reply:
This is nice. Let's please rename as

num_dataset_schema_invalid -> num_dataset_invalid_delta_schema
num_dataset_buggy_delta_schema -> num_dataset_valid_delta_schema

Reply:
addressed in https://github.com/datahub-project/datahub/pull/10299/commits/d634b10870bd0408cf3e5497c235bb5f0c4070a8